### DISCUSSION TITLE
M1 Mac Homebrew Installing Error

### DISCUSSION THREAD

**Discussion by nothehecat** [Homebrew/discussions#384] [Getting started] [answered]
Hey guys,

I installed homebrew on my new m1 mac using the instructions. However, I got an error message Warning: /opt/homebrew/bin is not in your PATH. When I try to enter in brew help, my command line gives me the following output: zsh: command not found: brew.

Can someone assist me with this? Thanks

**Comment by carlocab** [comment, marked as answer]
You need to add something like

    export PATH="/opt/homebrew/bin:$PATH"

to your `.zshrc` or `.bashrc` (or another equivalent, depending on the shell you're using).

**Reply by nothehecat** [reply]
Fixed - thanks

**Reply by CodingforMegan2011** [reply]
Fixed!

Thanks!

**Reply by Chadeg1** [reply]
How do I go about adding that?

**Comment by gregBerthelot** [comment]
Hi

`PATH` should be with or without quote? `export PATH="/opt/homebrew/bin:$PATH"` or `export PATH=/opt/homebrew/bin:$PATH` ?

Thanks

**Reply by carlocab** [reply]
With quotes is safer but without quotes should also work unless you (or one of your applications) has been putting weird things in your `PATH` (e.g. paths with spaces in them).

**Reply by alanspace** [reply]
Hello, where should export PATH="/opt/homebrew/bin:$PATH" be put? 


> # >>> conda initialize >>>
> # !! Contents within this block are managed by 'conda init' !!
> __conda_setup="$('/Users/username/opt/anaconda3/bin/conda' 'shell.zsh' 'hook' 2> /dev/null)"
> if [ $? -eq 0 ]; then
>     eval "$__conda_setup"
> else
>     if [ -f "/Users/username/opt/anaconda3/etc/profile.d/conda.sh" ]; then
>         . "/Users/username/opt/anaconda3/etc/profile.d/conda.sh"
>     else
>         export PATH="/Users/username/opt/anaconda3/bin:$PATH"
>     fi
> fi
> unset __conda_setup
> # <<< conda initialize <<<

**Reply by gregBerthelot** [reply]
You can add to your bash file `nano ~/.zshrc`
